ultravox/data/datasets.py

```diff
@@ -159,6 +159,11 @@ def __post_init__(self):
         if self.audio is not None:
             if self.audio.dtype == np.float64:
                 self.audio = self.audio.astype(np.float32)
+            elif self.audio.dtype == np.int16:
+                # Gradio mp3 file submission, as one example, seems to turn an
+                # mp3 into int16, thus it can be useful to convert to float32 if
+                # we come across this.
```

Comment:
Nit: I vote to remove the comment. It makes sense for the PR, but I don't think it makes as much sense once the PR has been merged.

Reply:
Thanks for explaining the rationale though, I was curious where you were hitting this.